Doc/howto/remote_debugging.rst

```diff
@@ -0,0 +1,335 @@
+.. _remote-debugging:
+
+Remote Debugging Attachment Protocol
+====================================
```

Comment:
Let's use sentence case for headers: https://devguide.python.org/documentation/style-guide/#capitalization

We're not very consistent, but let's at least follow it for new pages.

```suggestion
Remote debugging attachment protocol
====================================
```